# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1285922673 14400
#      Fri Oct 01 04:44:33 2010 -0400
# Node ID a22f3a673f3334c530d0b7c173574d618e4fe331
# Parent  44ff569a59a713591f9a41ae2bd6a28e68ca6cbd
mex.cc (mxArray_struct::as_octave_value): Use octave_map instead of Octave_map

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-10-01  John W. Eaton  <jwe@octave.org>
+
+	* mex.cc (mxArray_struct::as_octave_value): Use octave_map
+	instead of Octave_map.
+
 2010-10-01  John W. Eaton  <jwe@octave.org>
 
 	* ov-base.h: Delete forward decl for Octave_map class.
 
 2010-10-01  John W. Eaton  <jwe@octave.org>
 
 	* ov-struct.h (octave_struct (const Octave_map&)): Delete constructor.
 
diff --git a/src/mex.cc b/src/mex.cc
--- a/src/mex.cc
+++ b/src/mex.cc
@@ -1826,17 +1826,17 @@ public:
 protected:
 
   octave_value as_octave_value (void) const
   {
     dim_vector dv = dims_to_dim_vector ();
 
     string_vector keys (fields, nfields);
 
-    Octave_map m;
+    octave_map m;
 
     mwSize ntot = nfields * get_number_of_elements ();
 
     for (int i = 0; i < nfields; i++)
       {
         Cell c (dv);
 
         octave_value *p = c.fortran_vec ();
